client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/model/TaxiDto.kt

```diff
@@ -14,11 +14,11 @@ data class TaxiDto(
     @SerialName("type")
     val type: String?,
     @SerialName("driverId")
-    val driverId: String?,
+    val driverId: String? = null,
     @SerialName("seats")
     val seats: Int?,
-    @SerialName("username")
-    val username: String?,
+    @SerialName("driverUsername")
```

Comment:
the fields of driver and taxiInfo  can be encapsulated to taxiInfoDto and driverDto